common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -38,15 +38,20 @@
 public class WynncraftButtonFeature extends Feature {
     private static final String GAME_SERVER = "play.wynncraft.com";
     private static final String LOBBY_SERVER = "lobby.wynncraft.com";
+    private static boolean initialBoot = false;
```

Comment:
```suggestion
    private boolean initialBoot = false;
```

Feature variables musn't be static